src/xcode/ENA/ENA/Source/Models/Metadata/Test Result Metadata/TestResultMetadata.swift

```diff
@@ -5,11 +5,17 @@
 import Foundation
 
 struct TestResultMetadata: Codable {
+
+	enum TestType: Int, Codable {
```

Comment:
We already have a `CoronaTestType` enum, just mentioning. I guess there are pros and cons for using it here as well vs. adding a separate one.